src/core/ui/context/appearance.tsx

```diff
@@ -30,12 +30,20 @@ type AppearanceProviderProps = React.PropsWithChildren<SlashAuthStyle>;
 
 const AppearanceProvider = (props: AppearanceProviderProps) => {
   const ctxValue = useDeepEqualMemo(() => {
+    const theme = {
+      ...DEFAULT_MODAL_STYLES,
+      ...props.signInModalStyle,
+      testColor: 'red',
+    };
+
+    const root = document.documentElement;
+    Object.keys(theme).forEach((key) => {
+      root.style.setProperty(`--slashauth-${key}`, theme[key]);
+    });
```

Comment:
By doing this we are using native css variables with css-modules. This lets us handle :hover and other states in a better ways, as well as light and dark mode in case we want to give support.

cc/ @nedrocks